test/basic.js

```diff
@@ -1,36 +1,155 @@
-var test = require('tape')
-var ram = require('random-access-memory')
-var kappa = require('..')
+const tape = require('tape')
+const { Kappa } = require('..')
+const { runAll } = require('./lib/util')
 
-test('simple view', function (t) {
-  var core = kappa(ram, { valueEncoding: 'json' })
+tape('simple source', t => {
+  const kappa = new Kappa()
 
-  var sum = 0
+  kappa.use('view1', createSimpleSource(), createSimpleView())
+  kappa.use('view2', createSimpleSource(), createSimpleView())
+  kappa.source.view1.push(1)
+  kappa.source.view1.push(2)
+  kappa.source.view2.push(3)
+  kappa.source.view2.push(4)
```

Comment:
This is a bigger convo, but the API format of

`kappa.source.flowName.METHOD`

might feel nicer as

`kappa.flowName.source.METHOD`

What do you think? This is kinda a big change at this point, but it'll be an even bigger pain to make later on once even more downstreamers are using this API.